Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/IndentationStyleConfiguration.swift

```diff
@@ -0,0 +1,61 @@
+import SwiftLintCore
+
+struct IndentationStyleConfiguration: SeverityBasedRuleConfiguration, Equatable {
+	typealias Parent = IndentationStyleRule
+
+	@ConfigurationElement(key: "severity")
+	private(set) var severityConfiguration = SeverityConfiguration<Parent>.warning
+	@ConfigurationElement(key: "include_multiline_strings")
+	private(set) var includeMultilineStrings = false
+	@ConfigurationElement(key: "include_multiline_comments")
+	private(set) var includeMultilineComments = true
```

Comment:
If I recall correctly, it's because Xcode automatically indents multiline comments content with an additional space, even if you use tabs for indentation. As a tabs evangelist, even I don't always care to fix that since the smallest tweak can cause Xcode to reset that.

Reply:
If trailing spaces are only used to refine the indentation, but otherwise the file's default is respected by Xcode so that either spaces or tabs are used for the main chunk of the indentation, this seems like a normal case that should be supported anyway also in other areas, e.g.

```swift
if cond1,
   cond2 {
    // ...
}
```

wherein `cond2` might be indented by one tab and one space.

In other words, is there any reason to make this optional?